### DISCUSSION TITLE
Skid Row Lemmings intro works at half speed

### DISCUSSION THREAD

**Discussion by pik33** [BlitterStudio/amiberry#1206] [Q&A] [answered]
I installed a fresh Amiberry 5.6.5 on a RPi5/64bit Bookworm. The demo (Lemmings crack screen) runs at half speed.

The emulator is configured as standard A500 with kickstart 1.3

To check if it is not a Wayland problem I switches the system to X. No change.
SDL is reported as 2.26.5. 
The screen is configured at 1920x1080@50 Hz. I tried to reduce the resolution to 1280x800 and changre the refresh frequency. No success.

If I set the chipset emulation to cycle exact, the emulation speed returns to normal, however the sine scroller is messed up. 

I tried to downgrade to 5.6.3, and 5.6.2, no change.

I have also old 4.1.4 version. It works, the demo runs at the full speed and the scroller is not distorted.

**Comment by solskogen** [comment, marked as answer]
Confirmed. A workaround is to put blitter to "Immediate"

**Comment by pik33** [comment]
The workaround works.
